data/type.yaml.tmpl

```diff
@@ -3,6 +3,8 @@
 {% if prof == profile %}
 {% if profiles[profile]['steps'] %}
 classes:
+  - cloud
```

Comment:
we need to keep control on what can be paralleled so I don't like putting some classes here.